docs/data/external-oasis-repos.md

```diff
@@ -54,7 +56,17 @@ root@host # echo "cvmfs_server mount -a" >>/etc/rc.local
 root@host # chmod +x /etc/rc.local
 ```
 
-In addition, apache should listen on port 8000, have KeepAlive enabled, and be started. Use commands like these:
+For a RedHat EL6-based system:
```

Comment:
We should really split this, the EL7, and HTTPD instructions into subsections with numbered lists